plugins/woocommerce/tests/e2e-pw/utils/editor.js

```diff
@@ -1,14 +1,13 @@
 const { expect } = require( '@playwright/test' );
 
-const closeWelcomeModal = async ( { page } ) => {
-	// Close welcome popup if prompted
-	try {
-		await page
-			.getByLabel( 'Close', { exact: true } )
-			.click( { timeout: 5000 } );
-	} catch ( error ) {
-		// Welcome modal wasn't present, skipping action.
-	}
+const closeChoosePatternModal = async ( { page } ) => {
```

Comment:
I'm not sure the new locator handler code qualifies to be a util function, but I'm ok either way.